config/routes.rb

```diff
@@ -2,6 +2,9 @@
   root "users#index"
 
   resources :users, only: [:index, :show] do
-  resources :posts, only: [:index, :show]
+  resources :posts do
+    resources :comments 
+    resources :likes
```

Comment:
- [ ] **Optional:** Like can only be created, read and destroyed, in this project, you are only required to create it. So you could limit the routes to these only like what you did in line 4 above with the user's route.

Reply:
Thanks alot 🙌